Rakefile

```diff
@@ -2,7 +2,16 @@
 
 require 'rake'
 require 'rake/testtask'
-require 'bundler/gem_tasks'
+require "rake/clean"
+CLOBBER.include "pkg"
```

Comment:
Can you please explain why this CLOBBER is used?

Reply:
`bundler/gem_tasks` brings in several tasks for easier management of the gem (building, releasing, clearing temporary files...). Given there are now two gems living in the repository, I had to split those out. This lead to namespacing the gem building/releasing tasks and then defining the temporary-file-cleaning task (`CLOBBER`) by hand.